test/e2e/case1_pod_handling_test.go

```diff
@@ -293,4 +295,66 @@ var _ = Describe("Test pod obj template handling", Ordered, func() {
 			utils.KubectlDelete("event", "--field-selector=involvedObject.name="+configPolicyName, "-n", "managed")
 		})
 	})
+
+	Describe("Test pod obj in "+ocmNs+" namespace", Ordered, func() {
```

Comment:
Could you modify the existing testcase instead since if it runs at the same time as this test, the pod it creates will conflict.

Reply:
This is Ordered so it won't conflict

Reply:
```
Describe("Test pod obj template handling", Ordered, func() {
```